`PF2EquippedItemsComponent.cpp` Errors Prevent Creating Packaged/Cooked Builds Containing OpenPF2

Attempting to build for Windows on UE 5.3 yields:
```
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(48): error C2039: 'IsDataValid': is not a member of 'UPF2EquippedItemsComponent'
OpenPF2\Source\OpenPF2GameFramework\Public\Items\PF2EquippedItemsComponent.h(83): note: see declaration of 'UPF2EquippedItemsComponent'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(49): error C2270: 'IsDataValid': modifiers not allowed on nonmember functions
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(51): error C2653: 'Super': is not a class or namespace name
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(53): error C3861: 'ValidateSlots': identifier not found
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(54): error C3861: 'ValidateEquippedItems': identifier not found
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(237): error C2039: 'ValidateSlots': is not a member of 'UPF2EquippedItemsComponent'
OpenPF2\Source\OpenPF2GameFramework\Public\Items\PF2EquippedItemsComponent.h(83): note: see declaration of 'UPF2EquippedItemsComponent'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(238): error C2270: 'ValidateSlots': modifiers not allowed on nonmember functions
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(242): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(242): error C2530: 'Slot': references must be initialized
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(242): error C2143: syntax error: missing ';' before ':'
PackagingResults: Error: missing ';' before ':'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(242): error C2143: syntax error: missing ';' before ')'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(251): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
PackagingResults: Error: missing ';' before ')'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(262): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(276): error C2039: 'ValidateEquippedItems': is not a member of 'UPF2EquippedItemsComponent'
OpenPF2\Source\OpenPF2GameFramework\Public\Items\PF2EquippedItemsComponent.h(83): note: see declaration of 'UPF2EquippedItemsComponent'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(277): error C2270: 'ValidateEquippedItems': modifiers not allowed on nonmember functions
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(281): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(281): error C3536: 'EquippedItemIt': cannot be used before it is initialized
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(283): error C2100: illegal indirection
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(283): error C3617: initializers of structured bindings must be of array or non-union class type; type 'const int' is not permitted
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(287): error C2737: 'EntryIndex': const object must be initialized
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(290): error C2672: 'TArray<TSubclassOf<UPF2EquipableItemSlot>,FDefaultAllocator>::Contains': no matching overloaded function found
C:\Program Files\Epic Games\UE_5.3\Engine\Source\Runtime\Core\Public\Containers\Array.h(1185): note: could be 'bool TArray<TSubclassOf<UPF2EquipableItemSlot>,FDefaultAllocator>::Contains(const ComparisonType &) const'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(290): note: Failed to specialize function template 'bool TArray<TSubclassOf<UPF2EquipableItemSlot>,FDefaultAllocator>::Contains(const ComparisonType &) const'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(290): note: With the following template arguments:
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(290): note: 'ComparisonType=unknown-type'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(297): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(308): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(308): error C2530: 'CurrentSlot': references must be initialized
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(308): error C2143: syntax error: missing ';' before ':'
PackagingResults: Error: missing ';' before ':'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(308): error C2143: syntax error: missing ';' before ')'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(324): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
PackagingResults: Error: missing ';' before ')'
OpenPF2\Source\OpenPF2GameFramework\Private\Items\PF2EquippedItemsComponent.cpp(338): error C2355: 'this': can only be referenced inside non-static member functions or non-static data member initializers
```

--- a/Source/OpenPF2GameFramework/Private/Items/PF2EquipableItemSlot.cpp
+++ b/Source/OpenPF2GameFramework/Private/Items/PF2EquipableItemSlot.cpp
@@ -65,6 +65,7 @@ EDataValidationResult UPF2EquipableItemSlot::IsDataValid(FDataValidationContext&
 
 	return Result;
 }
+#endif
 
 FString UPF2EquipableItemSlot::GetIdForLogs() const
 {
@@ -77,6 +78,5 @@ FString UPF2EquipableItemSlot::GetIdForLogs() const
 		}
 	);
 }
-#endif
 
 #undef LOCTEXT_NAMESPACE

--- a/Source/OpenPF2GameFramework/Private/Items/PF2EquippedItemsComponent.cpp
+++ b/Source/OpenPF2GameFramework/Private/Items/PF2EquippedItemsComponent.cpp
@@ -45,6 +45,7 @@ UPF2EquippedItemsComponent::UPF2EquippedItemsComponent() : Events(nullptr)
 {
 }
 
+#if WITH_EDITOR
 EDataValidationResult UPF2EquippedItemsComponent::IsDataValid(FDataValidationContext& Context) const
 {
 	EDataValidationResult Result =
@@ -55,6 +56,7 @@ EDataValidationResult UPF2EquippedItemsComponent::IsDataValid(FDataValidationCon
 
 	return Result;
 }
+#endif
 
 void UPF2EquippedItemsComponent::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
 {
@@ -234,6 +236,7 @@ void UPF2EquippedItemsComponent::UnequipItemInSpecificSlot(const UPF2EquipableIt
 	}
 }
 
+#if WITH_EDITOR
 EDataValidationResult UPF2EquippedItemsComponent::ValidateSlots(FDataValidationContext& Context) const
 {
 	EDataValidationResult                      Result = EDataValidationResult::Valid;
@@ -343,6 +346,7 @@ EDataValidationResult UPF2EquippedItemsComponent::ValidateEquippedItems(FDataVal
 
 	return Result;
 }
+#endif
 
 void UPF2EquippedItemsComponent::OnRep_EquippedItems(const TArray<FPF2EquippedItem>& PreviouslyEquippedItems)
 {

--- a/Source/OpenPF2GameFramework/Public/Items/PF2EquipableItemSlot.h
+++ b/Source/OpenPF2GameFramework/Public/Items/PF2EquipableItemSlot.h
@@ -16,7 +16,7 @@ class IPF2ItemInterface;
  * arms; multiple ring slots for each finger; and a slot for a cloak.
  */
 UCLASS(Abstract, BlueprintType, Blueprintable)
-class UPF2EquipableItemSlot : public UDataAsset, public IPF2LogIdentifiableInterface
+class OPENPF2GAMEFRAMEWORK_API UPF2EquipableItemSlot : public UDataAsset, public IPF2LogIdentifiableInterface
 {
 	GENERATED_BODY()
 

--- a/Source/OpenPF2GameFramework/Public/Items/PF2EquippedItemsComponent.h
+++ b/Source/OpenPF2GameFramework/Public/Items/PF2EquippedItemsComponent.h
@@ -214,7 +214,7 @@ protected:
 	 */
 	virtual void UnequipItemInSpecificSlot(const UPF2EquipableItemSlot* Slot);
 
-	#if WITH_EDITOR
+#if WITH_EDITOR
 	/**
 	 * Validates that the SupportedSlots property contains valid data from the editor.
 	 *
@@ -242,7 +242,7 @@ protected:
 	 *	Whether the equipped items are valid.
 	 */
 	EDataValidationResult ValidateEquippedItems(FDataValidationContext& Context) const;
-	#endif
+#endif
 
 	// =================================================================================================================
 	// Protected Replication Callbacks

